app/src/main/java/org/mozilla/fenix/components/toolbar/BrowserToolbarView.kt

```diff
@@ -99,7 +99,10 @@ class BrowserToolbarView(
                     clipboard.text = selectedSession?.url
                 }
 
-                FenixSnackbar.make(view, Snackbar.LENGTH_SHORT)
+                FenixSnackbar.makeWithToolbarPadding(
+                    view.rootView.findViewById(android.R.id.content),
```

Comment:
are we able to just do `view.rootView` here? My understanding of how findViewById(android.R.id.content) works is that it just finds the root element of the view, but that should just be `view.rootView` no?

Reply:
No. `view.rootview` returns the DecorView. This is the topmost view in the current view hierarchy (it does not have an ID).
And think as `android.R.id.content` is just the view that contains whatever elements we put, not the system.
All these can be visually differentiated using layout inspector.

Reply:
Yep, you're right @mcarare, view.rootview returns the DecorView. 

I'm thinking, since android.R.id.content gives us the ContentFrameLayout (which gets changed with each activity), why not go one step further, and use the FrameLayout which sits on top (grandparent of content). This way, we avoid overlapping the navigation bar, but we preserve the snackbar between activities, so it can be used in a general way.